build/build-viewer.js

```diff
@@ -113,8 +113,15 @@ async function compileJs() {
   const generatorFilename = `${sourceDir}/../lighthouse-core/report/report-generator.js`;
   const generatorBrowserify = browserify(generatorFilename, {standalone: 'ReportGenerator'})
     .transform('brfs');
-  const generatorBundle = promisify(generatorBrowserify.bundle.bind(generatorBrowserify));
-  const generatorJs = (await generatorBundle()).toString();
+
+  /** @type {Promise<string>} */
+  const generatorJsPromise = new Promise(resolve => {
+    generatorBrowserify.bundle((err, src) => {
+      if (err) throw err;
+      resolve(src.toString());
+    });
+  });
+  const generatorJs = await generatorJsPromise;
```

Comment:
try as I might, I can't convince the compiler that the `promisify` call was working, so just made this an explicit promise

Reply:
i hacked my way around it

![image](https://user-images.githubusercontent.com/4071474/49260866-5e08b980-f3f4-11e8-9610-341fb860cc43.png)

seems TS doesn't like callback style functions having no errors ... which is very weird. see `promisify` in the standard node ts lib. None of them have a nullable Error